otherlibs/threads/thread.mli

```diff
@@ -40,7 +40,8 @@ val self : unit -> t
 val id : t -> int
 (** Return the identifier of the given thread. A thread identifier
    is an integer that identifies uniquely the thread.
-   It can be used to build data structures indexed by threads. *)
+   It can be used to build data structures indexed by threads.
+   @since 4.03 *)
```

Comment:
This annotation is not quite right, `Thread.id` was added by ee63e8d356b5717a32b0739fbe957f8e92d6e2c1 back in 1996.